### PR TITLE
add scripts to automate the setup and teardown of the local example

### PR DESCRIPTION
**Pull request by fgimenez**
The setup script is specific for minikube, maybe this could be configured.

### CODE REVIEW

File: examples/local/env.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+export CLUSTER_NAME=g8s
+export COMMON_DOMAIN=${CLUSTER_NAME}.local
```
Comment (MarcelMue) [inline examples/local/env.sh]: I don't think this is always the case, `CLUSTER_NAME` and `COMMON_DOMAIN` are completely unrelated in the setup with `aws-operator` I believe.

File: examples/local/setup-minikube.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        -e 's|${CLUSTER_NAME}|'"${CLUSTER_NAME}"'|g' \
+        -e 's|${COMMON_DOMAIN}|'"${COMMON_DOMAIN}"'|g' \
+        -e 's|${VAULT_HOST}|'"${VAULT_HOST}"'|g' \
+        -e 's|${VAULT_TOKEN}|'"${VAULT_TOKEN}"'|g' \
+        ./$f > ./${f%.tmpl.yaml}.yaml
+done
+
+kubectl apply -f ./vault.yaml
+eval $(minikube docker-env)
+(
```
Comment (MarcelMue) [inline examples/local/setup-minikube.sh]: I like this 👍

File: examples/local/env.sh (hunk 1 above)
Comment (rossf7) [reply examples/local/env.sh]: There is a connection between `cert-operator` and `aws-operator`. The `COMMON_DOMAIN` used should match the one in `aws-operator`. This is so the certs are valid and match the Route 53 domains. 

It also means you can configure `kubectl` to connect to the cluster. I'll update the wiki page to also include `cert-operator` as I missed this step.

File: examples/local/setup-minikube.sh (hunk 2 above)
Comment (rossf7) [reply examples/local/setup-minikube.sh]: Same here! 👍

File: examples/local/env.sh (hunk 1 above)
Comment (fgimenez) [reply examples/local/env.sh]: Ok thank you both, I've updated `COMMON_DOMAIN` to not depend on `CLUSTER_NAME`, let me know if anything else is needed here

File: examples/local/setup-minikube.sh (hunk 2 above)
Comment (fgimenez) [reply examples/local/setup-minikube.sh]: Thanks! :)

**Comment by fgimenez** [comment]
I've added two additional changes for building the binary from the local tree and taking into account existing env vars, pls take a look

**Pull request merged**